services/motion/explore/explore.go

```diff
@@ -550,11 +578,15 @@ func (ms *explore) createMotionPlan(
 		return nil, err
 	}
 
-	// replace original base frame with one that knows how to move itself and allow planning for
 	if err := ms.frameSystem.ReplaceFrame(kb.Kinematics()); err != nil {
 		return nil, err
 	}
 
+	// replace original base frame with one that knows how to move itself and allow planning for
```

Comment:
it seems we are using replaceFrame twice?

Reply:
Nice catch, missed that on my last round of checks